Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -0,0 +1,157 @@
+@Library('NodeHelper') _
+import jenkins.model.Jenkins;
+import hudson.model.Computer;
+
+clones = [:]
+List<String> newMachines = new ArrayList<String>();
+
+/* Iterates over all the online nodes in Jenkins and
+ * prints contents of workspace folder along with
+ * the space they occupy. With exception of tmp directory
+ */
+node {
+    stage('Print_Space_Monitoring_Data') {
+        NodeHelper nodeHelper = new NodeHelper();
+
+        String projectLabel = params.projectLabel;
```

Comment:
If I don't define this parameter, or if I don't set it, will I iterate over all the nodes?

Reply:
It won't iterate over any nodes. That's on my todo for after I have checked in rest of the jobs